google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -406,4 +406,51 @@
     <className>com/google/cloud/spanner/AbstractLazyInitializer</className>
     <method>java.lang.Object initialize()</method>
   </difference>
+  
+  <!-- RPC Priority -->
+  <difference>
```

Comment:
This change will break customers and requires a major version bump. This is not optional.

Reply:
This is no longer a breaking change, as the breaking changes that were needed for this and other changes were grouped into a separate change to avoid having to bump the version multiple times. That change added `Options... options` varargs to the methods that need to support a variety of query / update / transaction options.